index.html

```diff
@@ -204,7 +204,6 @@
           Promise&lt;void&gt; abort();
           Promise&lt;boolean&gt; canMakePayment();
 
-          readonly attribute DOMString? paymentRequestId;
```

Comment:
Also, the constructor algorithm doesn't ever set this to null - so maybe it should not be nullable. It's also unclear what happens if paymentRequestId is set to just "" (the empty string). 

Should setting it to `null` be treated as a special case (i.e., treat null as undefined instead of being converted to the word "null")?